.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,8 +1,15 @@
 <!--
+    Thank you for contributing!
+
     ESLint adheres to the [JS Foundation Code of Conduct](https://js.foundation/community/code-of-conduct).
 -->
 
-**What is the purpose of this pull request? (put an "X" next to item)**
+#### Prerequisites checklist
```

Comment:
These were changed because `markdownlint` now warns on using emphasized text as headers (`MD036/no-emphasis-as-heading/no-emphasis-as-header`). This should look the the same to the end user.